Source/JavaScriptCore/assembler/MacroAssemblerX86_64.h

```diff
@@ -3038,15 +3038,65 @@ class MacroAssemblerX86_64 : public MacroAssemblerX86Common {
             m_assembler.vroundpd_rr(input, dest, RoundingType::TowardZero);
     }
 
-    void vectorTruncSat(SIMDInfo simdInfo, FPRegisterID input, FPRegisterID dest)
+    void vectorTruncSat(SIMDInfo simdInfo, FPRegisterID src, FPRegisterID dest, RegisterID tmp, FPRegisterID scratch1, FPRegisterID scratch2)
     {
-        ASSERT(scalarTypeIsFloatingPoint(simdInfo.lane));
-        ASSERT(simdInfo.signMode != SIMDSignMode::None);
+        ASSERT(supportsAVXForSIMD());
```

Comment:
We are about to replace `supportsAVXForSIMD` with `supportsAVX`, so let's use `supportsAVX`